src/main/java/hu/oe/nik/szfmv/automatedcar/AutomatedCar.java

```diff
@@ -35,5 +35,6 @@ private void calculatePositionAndOrientation() {
         var movingVector = virtualFunctionBus.powertrainPacket.getMovingVector();
         x += movingVector.getX();
         y += movingVector.getY();
+        setRotation(pt.getAutoSzoge());
```

Comment:
![](https://pics.me.me/english-motherfucker-do-you-speak-it-13596485.png)

Reply:
Got it!